mira/dkg/resources/extract_eiffel_ontology.py

```diff
@@ -0,0 +1,433 @@
+from rdflib import Graph
+from pathlib import Path
+import os
+import csv
+import requests
+from curies import Converter
+from rdflib.term import URIRef
+
+HERE = Path(__file__).parent
+ONTOLOGY_FILES_DIR = HERE / 'eiffel_ttl_files'
+RESULTS_DIR = HERE / 'sql_query_spreadsheet_results'
+ECV_KB_URL = ('https://raw.githubusercontent.com/benmomo/'
+              'eiffel-ontology/main/ontology/ecv-kb.ttl')
+EO_KB_URL = ('https://raw.githubusercontent.com/benmomo/'
+             'eiffel-ontology/main/ontology/eo-kb.ttl')
+SDG_SERIES_URL = ('https://raw.githubusercontent.com/benmomo/eiffel'
+                  '-ontology/main/ontology/sdg-kos-series-2019-Q2-G-01.ttl')
+SDG_GOAL_URL = ('https://raw.githubusercontent.com/benmomo/eiffel-ontology/'
+                'main/ontology/sdg-kos-goals-targets-indicators.ttl')
+
+
+class GraphObj:
+    def __init__(self, graph, name):
+        self.graph = graph
+        self.name = name
+        self.file_uri_list = []
+        self.info_list = []
+        self.relationship_list = []
+        self.class_list = []
+
+
+def get_store_graphs():
+    os.makedirs(ONTOLOGY_FILES_DIR, exist_ok=True)
+    os.makedirs(RESULTS_DIR, exist_ok=True)
+
+    ecv_kb_request = requests.get(ECV_KB_URL)
+    with open(str(ONTOLOGY_FILES_DIR) + '/' + 'ecv-kb.ttl', 'w') as file:
+        file.write(ecv_kb_request.text)
+
+    eo_kb_request = requests.get(EO_KB_URL)
+    with open(str(ONTOLOGY_FILES_DIR) + '/' + 'eo-kb.ttl', 'w') as file:
+        file.write(eo_kb_request.text)
+
+    sgd_series_request = requests.get(SDG_SERIES_URL)
+    with open(str(ONTOLOGY_FILES_DIR) + '/' +
+              'sdg-kos-series-2019-Q2-G-01.ttl',
+              'w') as file:
+        file.write(sgd_series_request.text)
+
+    sdg_goal_request = requests.get(SDG_GOAL_URL)
+    with open(str(ONTOLOGY_FILES_DIR) +
+              '/sdg-kos-goals-targets-indicators.ttl',
+              'w') as file:
+        file.write(sdg_goal_request.text)
+
+    graph_obj_map = {}
+    for file_name in os.listdir(ONTOLOGY_FILES_DIR):
+        content_path = 'eiffel_ttl_files/' + file_name
+        with open(content_path) as file:
+            graph_obj_ = GraphObj(graph=Graph(),
+                                  name=file_name[:len(file_name) - 4])
```

Comment:
when you use the `pathlib` interface, you get lots of nice high-level operations so you don't have to re-implement filename parsing yourself